src/main/java/homeworks/Infrastructure/Exersise_2_Test_Logger.java

```diff
@@ -7,12 +7,12 @@
 
     public static int counter = 1;
 
-    public String Log(String operation){
+    public void Log(String operation){
```

Comment:
rename to "log(String operation)" according to java convention

Reply:
done